app/assets/javascripts/actions/conference_actions.es6.jsx

```diff
@@ -86,6 +74,14 @@
       };
     }
 
+    storeListAttribute(key, value, index) {
```

Comment:
@maxwolffe how do you feel about refactoring this into the regular `storeAttribute` action?
The store would just check if an `index` is passed in to know what handle to do.